CHANGELOG.md

```diff
@@ -13,6 +13,30 @@ Older versions of this project were distributed as [pybigquery][0].
 
 [2]: https://pypi.org/project/pybigquery/#history
 
+## [1.11.0.dev3](https://github.com/googleapis/python-bigquery-sqlalchemy/compare/v1.9.0...v1.11.0.dev3) (2024-02-20)
```

Comment:
We should update this to be footers of the commit message, per: https://github.com/googleapis/release-please/blob/main/README.md#what-if-my-pr-contains-multiple-fixes-or-features